hugo/content/privacy/_index.md

```diff
@@ -2,14 +2,11 @@
 title: Privacy Policy
 ---
 
-### TODOs
+SciLifeLab operates the [genomes.scilifelab.se](https://genomes.scilifelab.se/) website, which provides the **Service**.
 
-1. Does this also need to be provided in Swedish?
-2. Someone other than Rory needs to check the content/wording etc...
+This page is used to inform website visitors regarding our personal data processing policy. If you choose to use our **Service**, then your personal data will be processed in accordance with this policy.
 
-### Privacy Policy Overview
-
-This site is operated by SciLifeLab Data Centre and this page is used to inform website visitors regarding our data processing and privacy policy. If you choose to use our **Service**, then your personal data will be processed in accordance with this policy.
+The personal information that we collect is used for providing and improving the **Service**. We will not use or share your information with anyone except as described in this policy. All collected personal information will be processed for research purposes, i.e. using the lawful basis of public interest and in accordance with Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016, the General Data Protection Regulation.
```

Comment:
The text on L9 is repeated L19-20. Perhaps this is a typo? A quick look at pathogens.se shows that they only have this text on their L9-equivalent spot.

Reply:
Thanks for spotting this, I've deleted the repeat in commit https://github.com/ScilifelabDataCentre/genome-portal/pull/57/commits/6df7a70e09aa2bde6268ef1d1c71d38993838d3f